mimalloc-sys-test/Cargo.toml

```diff
@@ -3,16 +3,10 @@ name = "mimalloc-sys-test"
 version = "0.1.0"
 authors = ["gnzlbg <gonzalobg88@gmail.com>"]
 edition = "2015"
-build = "build.rs"
 
 [dependencies]
 mimalloc-sys = { path = "../mimalloc-sys" }
 libc = "0.2"
 
 [build-dependencies]
 ctest = "0.2"
-
-[[test]]
-name = "main"
-path = "test/main.rs"
-harness = false
```

Comment:
Why is this test removed? Have you checked that the test is actually executed ? Note that the test harness should be manually disabled here.

Reply:
I based my changes on [flate2-rs](https://github.com/alexcrichton/flate2-rs)'s systest. It uses a main.rs for systest package and the CI script uses `cargo run` but I don't mind reverting.

Reply:
Ah this should work as well!